src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -106,6 +122,14 @@ public function handle(GetResponseEvent $event)
             if ($authenticatedToken = $this->authenticationManager->authenticate($token)) {
                 $this->securityContext->setToken($authenticatedToken);
 
+                //return response to redirect after successful login
+                $response = $this->successHandler->onAuthenticationSuccess($request, $authenticatedToken);
```

Comment:
I just took this code from Symfony\Component\Security\Http\Firewall\AbstractAuthenticationListener::onSuccess because it was used by symfony's default authentication routine I was stepping through in debugger. By doing this I achieved redirect after successful login, which I think is good for security reasons because it is really easy to expose you login details by just coping and then sending that link to colleague. It was necessary to generate a proper response to make remeber_me cookie work.